### DISCUSSION TITLE
如何设置表情包镜像？

### DISCUSSION THREAD

**Discussion by codejiangqihan** [walinejs/waline#1691] [Q&A]
![Screenshot_20221224_093401](https://user-images.githubusercontent.com/107793872/209416989-2dad8de5-9f19-47f2-b1c9-07e3fadb735b.jpg)
就是这个表情包，我觉得unpkg太慢了，想使用国内的unpkg镜像，但是没找到方法，应该要怎么弄？

**Comment by lizheming** [comment]
- 怎么在 Waline 设置表情：https://waline.js.org/guide/features/emoji.html 
- unpkg 地址真么替换成镜像地址：替换 unpkg.com 为镜像地址
- 怎么自架 unpkg 镜像：自行搜索一下

问问题之前先自行搜索一下吧，这个问题我相信搜索之后就能找到答案的
